regions:
- id: '1'
  center:
  - 30.945341614906834
  - -48.695466832344586
  points:
  - - 27.73913043478261
    - -41.20229913048122
  - - 26.982608695652175
    - -44.22838608700296
  - - 27.73913043478261
    - -47.2544730435247
  - - 29.63043478260869
    - -49.65012521743774
  - - 31.89999999999999
    - -51.91969043482904
  - - 34.8
    - -53.0544730435247
  - - 37.826086956521735
    - -53.55882086961166
  offset_lateral_m: 0.0
  offset_x_m: 1.0
  offset_y_m: 1.0
- id: s
  center:
  - -23.410144927536233
  - -32.460270144973975
  points:
  - - -13.23913043478261
    - -28.21534260874209
  - - -16.391304347826086
    - -28.34142956526383
  - - -20.047826086956526
    - -28.46751652178557
  - - -23.45217391304348
    - -28.46751652178557
  - - -26.35217391304348
    - -29.60229913048122
  - - -27.61304347826087
    - -32.37621217395948
  - - -27.86521739130435
    - -35.780560000046435
  - - -27.86521739130435
    - -38.80664695656818
  - - -27.86521739130435
    - -42.084907826133396
  offset_lateral_m: 0.0
  offset_x_m: 1.0
  offset_y_m: 2.0
description: Regions for geometric offset along global paths. offset_lateral_m is
  applied along lane normal; offset_x_m, offset_y_m are world X/Y shifts.
